interpreter/shared/src/main/scala/sigmastate/interpreter/InterpreterContext.scala

```diff
@@ -48,13 +45,17 @@ object ContextExtension {
       val extSize = r.getByte()
       if (extSize < 0)
         error(s"Negative amount of context extension values: $extSize")
-      val ext = (0 until extSize)
-        .map(_ => (r.getByte(), r.getValue().asInstanceOf[EvaluatedValue[_ <: SType]]))
-      ContextExtension(mutable.LinkedHashMap(ext:_*))
-    }
 
-  }
+      val values = (0 until extSize)
+          .map(_ => (r.getByte(), r.getValue().asInstanceOf[EvaluatedValue[_ <: SType]]))
 
+      // TODO v6.0: enforce ordering e.g. using the code below (see https://github.com/ScorexFoundation/sigmastate-interpreter/issues/681)
+      // if (VersionContext.current.activatedVersion > JitActivationVersion) {
```

Comment:
Such Versioning does not work here, as old clients will be broken after activation. Remove misleading TODO

Reply:
removed